lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -197,6 +198,8 @@ class EffectiveClangContext {
       DC = omDecl->getCanonicalDecl();
     } else if (auto fDecl = dyn_cast<clang::FunctionDecl>(dc)) {
       DC = fDecl->getCanonicalDecl();
+    } else if (auto externCDecl = dyn_cast<clang::LinkageSpecDecl>(dc)) {
+      DC = externCDecl->getLexicalDeclContext();
     } else {
       assert(isa<clang::TranslationUnitDecl>(dc) ||
              isa<clang::LinkageSpecDecl>(dc) ||
```

Comment:
Minor: if we are editing around here, this line can be removed.

(sorry for reviewing late)